azure-pipelines.yml

```diff
@@ -96,8 +96,8 @@ parameters:
       - name: 'rpts'
         shutter: 'default'
         subscription: DCD-CFTAPPS-PROD
-      - name: 'privatelaw'
-        shutter: 'privatelaw'
+      - name: 'privatel'
```

Comment:
Is current not more meaningful? Curious on why the change?

Reply:
Hi @thomast1906 , 

Definitely I ageree, but I have been trying to figure out why the shuttering is not working. I think the only reason is this when I go through the hmcts.github.io -> **Use the first 8 characters only, of the name attribute from the env.tfvar**

Reference: https://hmcts.github.io/cloud-native-platform/path-to-live/shutter.html#uploading-static-shutter-pages-to-azure-storage-account
Use the first 8 characters only, of the name attribute from the env.tfvar file e.g. If the name attribute in env.tfvars file is xui-manage-org then ${shuttername} will be xuimanag.

Reply:
ok - thanks for that